ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAP-620] Prevent legacy means_passport being reused

## Description of change
Form reset when key details are changed did not change the application `means_passport` back to an empty array.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-620

## Notes for reviewer
Existing behaviour did not reset means_passport value meaning any legacy values persist even if other key details e.g. name, nino, etc are changed.

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature
Create an application which has means_passport, change name details before submitting, after submitting means_passport should be blank (check database).
